app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_post_comments_counter
+  after_create :update_post_comments_counter
+  after_destroy :update_post_comments_counter
+
```

Comment:
- [ ]  I suggest the comment counter is only updated when a comment is saved to the database, updating after creating and destroying a comment is unnecessary :blush: